src/pipe.ts

```diff
@@ -5,7 +5,7 @@ import ReturnPipeType from "./types/ReturnPipeType";
 
 /**
  * Performs left to right function composition.
- * The first argument have any value; the remaining arguments must be unary.
+ * The first argument can have any value; the remaining arguments must be unary.
```

Comment:
I think In this case, it seems better to assert with 'have' without 'can'

Reply:
Does *any* in this context refer to TypeScript `any` type?

Reply:
> Does _any_ in this context refer to TypeScript `any` type?

It's not intended to be Typescript `any` type
I meant any data that could be used as a value

Reply:
I think `The first argument can have any value` or `The first argument can be given any types of value` seems fine, but if you think `The first argument have any value` is better, I can revert it.

What do you think?

Reply:
hmm... I thought a little more. both would be fine